test/IDE/print_ast_tc_decls.swift

```diff
@@ -1330,14 +1330,15 @@ protocol ProtocolToExtend {
 extension ProtocolToExtend where Self.Assoc == Int {}
 // PREFER_TYPE_REPR_PRINTING: extension ProtocolToExtend where Self.Assoc == Int {
 
+// Should be evaluated before type checking.
 #if true
 #elseif false
 #else
 #endif
-// PASS_PRINT_AST: #if
-// PASS_PRINT_AST: #elseif
-// PASS_PRINT_AST: #else
-// PASS_PRINT_AST: #endif
+// PASS_PRINT_AST-NOT: #if
```

Comment:
I don't think this is correct either. It's important to have the document structure in place for SourceKit and other tools.